- name: Camera Node Watchdog
  description: Monitors the camera driver node to ensure still publishing. If not, kills it.
  restart_timeout: 30
  monitors:
      - monitor_type: TopicAlive
        topic: /camera/camera_info
        max_duration: 10
  actions:
      - action_type: KillNode
        node_name: /camera_node
      - action_type: SendEmail
        to_addresses:  [a.vanzo@hw.ac.uk, c.dondrup@hw.ac.uk]
        from_address: pepper.ideapark@gmail.com
        message: "Hello World"
        server: smtp.gmail.com
        port: 587
